src/main/java/org/zalando/nakadi/service/CursorConverter.java

```diff
@@ -0,0 +1,44 @@
+package org.zalando.nakadi.service;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.zalando.nakadi.domain.NakadiCursor;
+import org.zalando.nakadi.util.FeatureToggleService;
+import org.zalando.nakadi.view.Cursor;
+import org.zalando.nakadi.view.SubscriptionCursor;
+
+@Service
+public class CursorConverter {
```

Comment:
@v-stepanov As we discussed, it will be better to implement it a little bit later